company_website/view_helpers.py

```diff
@@ -2,11 +2,21 @@
 from typing import Dict
 
 from django.views.generic import TemplateView
+from django.views.generic.base import ContextMixin
 
 from company_website.models import PageSeo
+from project_liberation.settings.production import GOOGLE_ADS_CONVERSION_ID
+from project_liberation.settings.production import GOOGLE_ADS_CONVERSION_TARGET_ADDRESS
 
 
-class CustomTemplateView(TemplateView):
+class GoogleAdsMixin(ContextMixin):
+    extra_context = {
+        "google_ads_conversion_id": GOOGLE_ADS_CONVERSION_ID,
+        "google_ads_conversion_target_address": GOOGLE_ADS_CONVERSION_TARGET_ADDRESS,
```

Comment:
I think that we still also pass it as capital letters instead of small ones. Please change it. These variables are still our settings and they should be easy to recognize

Reply:
Done